src/main/java/gdsc/konkuk/platformcore/filter/auth/CustomAuthenticationFailureHandler.java

```diff
@@ -25,10 +24,8 @@ public class CustomAuthenticationFailureHandler extends SimpleUrlAuthenticationF
     public void onAuthenticationFailure(HttpServletRequest request, HttpServletResponse response,
             AuthenticationException exception) throws
             IOException {
-
-        CustomErrorCode errorCode =
-                request.getContentType().startsWith(APPLICATION_FORM_URLENCODED_VALUE)
-                        ? MemberErrorCode.INVALID_USER_INFO : GlobalErrorCode.ARGUMENT_NOT_VALID;
+        log.info("Authentication failed: {}", exception.getMessage());
+        CustomErrorCode errorCode = MemberErrorCode.INVALID_USER_INFO;
         ErrorResponse errorResponse = ErrorResponse.of(errorCode);
```

Comment:
```suggestion
        ErrorResponse errorResponse = ErrorResponse.of(MemberErrorCode.INVALID_USER_INFO);
```
분기가 없다면 바로 리턴하는게 어때요?

Reply:
좋습니다!